docs/logql.md

```diff
@@ -1,26 +1,28 @@
 # LogQL: Log Query Language
 
 Loki comes with its own PromQL-inspired language for queries called *LogQL*.
-LogQL can be considered a distributed `grep` that aggregates log sources and
+LogQL can be considered a distributed `grep` that aggregates log sources,
 using labels and operators for filtering.
 
-There are two types of LogQL queries: *log queries* which return the contents of
-log lines, and *metric queries* which extend log queries and calculates values
-based on the counts of logs from a log query.
+There are two types of LogQL queries: 
 
-A basic log query consists of two parts: the **log stream selector** and a
-**filter expression**. Due to Loki's design, all LogQL queries are required to
-contain a log stream selector.
+-*log queries* return the contents of log lines.
+-*metric queries* extend log queries and calculate values based on the counts of logs from a log query.
+
+A basic log query consists of two parts: 
+-**log stream selector**
```

Comment:
I am confused by your use of bold vs. italic. I recommend reviewing the [Microsoft Style Guide](https://docs.microsoft.com/en-us/style-guide/welcome/) and the [Grafana documentation style guide ](https://github.com/grafana/grafana/blob/master/contribute/style-guides/documentation-style-guide.md)for an idea of how we are using it in that project.

Reply:
Fab. Thank you!